Cargo.toml

```diff
@@ -155,6 +155,7 @@ rust.unreachable_pub = "warn"
 rust.unused_must_use = "deny"
 rust.rust_2018_idioms = { level = "deny", priority = -1 }
 rustdoc.all = "warn"
+rust.unnameable-types = "warn"
```

Comment:
If we are going to solve these crate-by-crate, we have to remove this:
```suggestion
```

And instead apply the lint just on the `net` crates, otherwise it will not be possible to merge this until fixes for _all_ crates are applied